tests/integration/adapter/rest-adapter-test.js

```diff
@@ -1700,8 +1700,11 @@ test("findHasMany - returning an array populates the array", function(assert) {
 });
 
 test("findHasMany - passes buildURL the requestType", function(assert) {
+  assert.expect(2);
   adapter.shouldBackgroundReloadRecord = () => false;
+  let optionsStub = { options: true };
   adapter.buildURL = function(type, id, snapshot, requestType) {
+    assert.equal(snapshot.adapterOptions, optionsStub);
```

Comment:
An assertion that the `snapshot` is instance of `DS.Snapshot` is sufficient here. There is actually no way to pass `adapterOptions` here, so that's not needed.